src/elements/Accordion/Accordion.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Accordion
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import Grid from '../Grid/Grid';
+import { isMobile } from '../../util/helpers';
+
+class Accordion extends Component {
+	static propTypes = {
+		children: PropTypes.node.isRequired,
+		title: PropTypes.node.isRequired,
+		className: PropTypes.string,
+		expanded: PropTypes.bool,
+		notification: PropTypes.bool // this is used to style accordion as a notification
+	};
+
+	constructor(props) {
+		super(props);
+
+		if (this.props.expanded) {
+			this.state = {
+				showContent: true
+			};
+		} else {
+			this.state = {
+				showContent: false
+			};
+		}
+
+		this.toggleAccordion = this.toggleAccordion.bind(this);
+	}
+
+	toggleAccordion() {
+		this.setState({
+			showContent: !this.state.showContent
+		});
+	}
+
+	render() {
+		const { title, children, className, notification } = this.props,
+			{ showContent } = this.state,
+			itemWidths = showContent && !isMobile() && notification ? [3, 7, 2] : [10, 2],
+			toggleText = showContent ? 'hide' : 'show',
+			toggleIconClass = showContent ? 'arrow arrow-up' : 'arrow arrow-down';
+
+		const combinedClasses = classNames(
+			'ui-accordion',
+			this.props.notification && 'notification',
+			className
+		);
+
+		return (
+			<Div className={combinedClasses}>
```

Comment:
This should be just a normal `div`.

The `Div` component is only needed when either:

A) You're going to be using one of the common methods/properties on the UI component version (e.g. `<Div marginTop="medium">`)

or B) You're using it as a child to a `<Grid>` component, which needs React components as children in order to apply all of the necessary properties when setting up on the grid items.